### DISCUSSION TITLE
My nested graphql query is not working. Any Ideas how to solve it?

### DISCUSSION THREAD

**Discussion by ezraermy** [Akryum/vue-apollo#1330] [Q&A] [answered]
<template>
  <div class="hello">
    <div v-for="user in users" :key="user.id">{{user.email}}</div>
  </div>
</template>

<script>
import gql from "graphql-tag"
export default {
  name: 'HelloWorld',
  props: {
    msg: String
  },
  apollo: {
    users: gql`
    query {
  users {
    id
    username
    email
    dateJoined
  }
}`
  }
}
</script>

// The above query works fine. However when I change the query object as shown below, It didn't work. Any help?

<template>
  <div class="hello">
    <div v-for="book in books" :key="book.id">{{book.title}}</div>
  </div>
</template>

<script>
import gql from "graphql-tag"
export default {
  name: 'HelloWorld',
  props: {
    msg: String
  },
  apollo: {
    books: gql`
    query {
  books {
    edges {
      node {
        id
        author {
          firstName
          lastName
        }
        title
        summary
        reviewSet {
          user {
            username
            email
            dateJoined
          }
          value
        }
      }
    }
  }
}`
  }
}
</script>

**Comment by Akryum** [comment, marked as answer]
You probably need to do `{{ books.edges[0].node.title }}`

**Reply by ezraermy** [reply]
Thanks a lot Akryum. It works fine now.
